react-native/react/actions/config/index.js

```diff
@@ -19,20 +19,27 @@ export function startup () {
         dispatch({type: Constants.startupLoaded, payload: error, error: true})
         return
       }
-      engine.rpc('config.getCurrentStatus', {}, incomingMap, (error, status) => {
-        if (error) {
-          dispatch({type: Constants.startupLoaded, payload: error, error: true})
-          return
-        }
-        dispatch({
-          type: Constants.startupLoaded,
-          payload: {config, status}
+
+      function getCurrentStatus () {
```

Comment:
might be nicer for this to be another action, then this just dispatches it (same with onconnect)